### DISCUSSION TITLE
Kaggle

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#497] [General]
https://www.kaggle.com/
這一樣是個很不錯的平台
他可以作競賽 裡面也有課程可以參考
更重要的是 他競賽裡 每個人傳的程式碼 都可以瀏覽參考 學習喔

**Comment by lx3211** [comment]
我也覺得這個平台很棒！
可以跟好幾萬人一起非同步比賽
而且裡面的data對於訓練其他模型也是很好的參考資料來源

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by xCk27x** [comment]
大部分的功能都是免費的 超棒🎉
但好像沒有提供教學影片輔助學習

**Comment by williamyen0623** [comment]
對於沒有經濟能力的學生來說
免費的學習資源真的是相當珍貴
還有競賽內容 感覺就很熱血 有種同儕一起學習的感覺

**Comment by limitx0** [comment]
這個網站裡面有資料有教學有比賽超棒的
而且有些比賽還有獎金

**Comment by skyling0299** [comment]
Kaggle是一個數據建模和數據分析競賽平台。企業和研究者可在其上發布數據，統計學者和數據挖掘專家可在其上進行競賽以產生最好的模型。這一眾包模式依賴於這一事實，即有眾多策略可以用於解決幾乎所有預測建模的問題，而研究者不可能在一開始就了解什麼方法對於特定問題是最為有效的。(測試用)
